certified-connectors/HoneywellForge/apiDefinition.swagger.json

```diff
@@ -0,0 +1,432 @@
+                    "format": "int32",
+                    "description": "The date and time when the fault has occurred in UNIX epoch format.",
+                    "title": "Fault Creation Timestamp"
+                  },
+                  "serviceCaseNumber": {
+                    "type": "string",
+                    "description": "The unique number that identifies the service case.",
+                    "title": "Service Case Number"
+                  },
+                  "siteId": {
+                    "type": "string",
+                    "description": "The ID of the site where the equipment that caused the service case is located.",
+                    "title": "Site ID"
+                  },
+                  "location": {
+                    "type": "string",
+                    "description": "The specific location within a site where the equipment that caused the service case is located. This can be a building, a floor, or a zone.",
+                    "title": "Equipment Location"
+                  },
+                  "equipmentGroup": {
+                    "type": "string",
+                    "description": "The type of the equipment that caused the service case, such as AHU, VAV.",
+                    "title": "Equipment Type"
+                  },
+                  "status": {
+                    "type": "string",
+                    "description": "The status of the service case.",
+                    "title": "Status"
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "The webhook has been registered."
+          }
+        },
+        "summary": "Register a webhook that allows you to receive open service cases",
+        "consumes": [
+          "application/json"
+        ],
+        "description": "This operation registers a webhook that allows you to receive open service cases from Honeywell Forge.",
+        "operationId": "ServiceCaseCreated",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "projectId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Specify an ID for your project. The Project ID is a unique identifier that represents you as a customer in Honeywell Forge.",
+            "x-ms-summary": "Project ID",
+            "x-example": "dynamics-azure",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "connectorId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Specify an ID for your connector. Connector ID is a unique identifier that represents your connector in Honeywell Forge.",
+            "x-ms-summary": "Connector ID",
+            "x-example": "e533b6fa-d7ee-4bed-bf30-02f433df86e6",
```

Comment:
@ashishdotme Secret-look-a-like values are not allowed in any connector artifacts, please remove them. If you can not please provide business justification.

Applies same to similar values in the connector files